lerna.json

```diff
@@ -5,6 +5,7 @@
   "useWorkspaces": true,
   "command": {
     "publish": {
+      "ignoreChanges": ["**/stories/**", "**/__tests__/**"],
```

Comment:
preventing lerna from thinking changes to tests and stories warrant a publish of source code.